larq_compute_engine/tests/end2end_test.py

```diff
@@ -152,35 +145,45 @@ def toy_model_int8(**kwargs):
 
 
 def preprocess(data):
-    return lqz.preprocess_input(data["image"])
+    return lqz.preprocess_input(data["image"]), data["label"]
 
 
 def assert_model_output(model_lce, inputs, outputs):
     interpreter = Interpreter(model_lce, num_threads=min(os.cpu_count(), 4))
     actual_outputs = interpreter.predict(inputs)
-    np.testing.assert_allclose(actual_outputs, outputs, rtol=0.001, atol=0.25)
+    np.testing.assert_allclose(actual_outputs, outputs, rtol=0.05, atol=0.125)
```

Comment:
I don't think it really made sense to have such a small rtol but huge atol (with rtol/atol 0.001/0.25, the rtol would only really be relevant for values with absolute value > 100), so I've bumped up the rtol as well as reducing the atol.

Reply:
I agree that `0.25` is huge. However it's what we observed for the keras-tflite difference for large models like Quicknet.
I think it might be better to pass a tolerance (or `Default`) to each model in `model_cls` so that we have normal tolerances everywhere and make it higher exclusively for QuickNet. What do you think?

Reply:
Yeah I agree, that's a good idea :+1: